### PR TITLE
Revising documentation

### PR DESCRIPTION
**Pull request by vezdeneszter**
## Type Of Change

- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Any modification in the test cases
- [x] Any modification in the documentation

### Checklist:

- [x] I have read and followed the [contribution guideline](https://github.com/axem-solutions/.github/blob/main/CONTRIBUTING.md).
- [x] I have checked to ensure there aren't other open Pull Requests for the same update/change.
- [x] All the test cases pass and new modifications in the production code are 100% covered.
- [x] I have made corresponding changes to the documentation.

## Related Issue
Closing: [DEM-263](https://axem.atlassian.net/browse/DEM-263)

## Description
Documentation changes:

- Added new format
- Re-wrote introduction page
- Added Discord CTA
- Re-arranged Commands page

[DEM-263]: https://axem.atlassian.net/browse/DEM-263?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/commands.md [hunk 1]
```diff
@@ -1,61 +1,74 @@
...
+
+#Commands
 
-- Development Environment management
-- Development Environment Catalog management
-- Registry management
-- Host management
+!!! question "Bug report"
+
+    We encourage you to join our open-source community and use DEM.
```
Comment (janosmurai) [inline docs/commands.md:9]: Remove "and use DEM"

File: docs/commands.md [hunk 2]
```diff
@@ -74,36 +87,46 @@ When the Dev Env is ready, press :material-keyboard-return: on the `save` button
...
 ## **`dem export DEV_ENV_NAME [PATH_TO_EXPORT]`**
 
-Export a Development Environment descriptor in JSON format to a text file. This file can be imported with the `load` command on another host. 
+**Description:**
 
-The way the file gets named can be set by the PATH_TO_EXPORT argument:
+Export a Development Environment descriptor in JSON format to a text file. 
 
-1. If it's not set, the file gets saved to the current directory with the name of the Development 
+This file can be imported with the `load` command on another host.
```
Comment (janosmurai) [inline docs/commands.md:119]: `load` -> `import` command (my bad)

File: docs/commands.md [hunk 3]
```diff
@@ -289,118 +360,163 @@ argument is the name of the Development Environment.
...
+
 The name of the registry is what you would normally use to pull an image.
 Examples:
 
-- If the full image tag is: repository/image:tag -> the name should be repository.
-- If the full image tag is: 192.168.1.1:5000/image:tag -> the name should be 192.168.1.1:5000
+
+| Full image tag                   | Name              |
+|----------------------------------|-------------------|
+| `repository/image:tag`           | a repository      |
```
Comment (janosmurai) [inline docs/commands.md:447]: "a repository" -> "repository"

File: docs/index.md [hunk 4]
```diff
@@ -1,69 +1,88 @@
+
 # Introduction
 
-**DEM** (Development Environment Manager) is an open source command line tool to **manage 
-containerized Development Environments.**
+## **What is DEM?**
+
+DEM (Development Environment Manager) is an **open-source command line tool for setting up, managing, and sharing Development Environments** for software teams.
```
Comment (janosmurai) [inline docs/index.md:6]: "for software teams" -> not needed

File: docs/index.md [hunk 5]
```diff
@@ -1,69 +1,88 @@
...
+
+DEM empowers developers to **create, modify, and maintain consistent Development Environments with ease**, bridging the gap between code creation and deployment.
```
Comment (janosmurai) [inline docs/index.md:8]: ", bridging the gap between code creation and deployment"
I don't understand what this means.

File: docs/index.md [hunk 6]
```diff
@@ -1,69 +1,88 @@
...
+
+With DEM, you can manage your development infrastructure just as you manage your codebase. By adopting DEM's methodologies for environment setup and management, you can significantly reduce context switching and the time between ideation and execution.
```
Comment (janosmurai) [inline docs/index.md:10]:
```suggestion
With DEM, you can manage your development infrastructure and codebase with the same version control system. By adopting DEM's methodologies for environment setup and management, you can significantly reduce the time needed for context switching.
```

File: docs/index.md [hunk 7]
```diff
@@ -1,69 +1,88 @@
...
+- DEM ensures **quick** and **simple installation**, and allows for **reinstallation whenever needed**.
+
+!!! linux_support "DEM for Linux"
+    Linux operating system is fully supported.
+
+    Check out the [Installation](https://www.axemsolutions.io/dem_doc/latest/installation/) section and feel free to ask any questions
+    in our community through [Discord](https://discord.gg/3aHuJBNvrJ).
+
+!!! windows_support "DEM for Windows"
+    Coming soon...
```
Comment (janosmurai) [inline docs/index.md:71]:
```suggestion
    DEM can be used in Windows Subsystem for Linux (WSL).
    
    Native Windows support is coming soon!
```

**Review by janosmurai — CHANGES_REQUESTED** [review]

File: docs/index.md (hunk 6 above)
Comment (vezdeneszter) [reply docs/index.md:10]: Removed fort sentence. modified second.

**Pull request merged**